.evergreen/config-atlas.yml

```diff
@@ -420,6 +420,20 @@ functions:
           ${PREPARE_SHELL}
           SERVERLESS=1 SSL=ssl RVM_RUBY="${RVM_RUBY}" SINGLE_MONGOS="${SINGLE_MONGOS}" MULTI_ATLASPROXY_SERVERLESS_URI="${MULTI_ATLASPROXY_SERVERLESS_URI}" SINGLE_ATLASPROXY_SERVERLESS_URI="${SINGLE_ATLASPROXY_SERVERLESS_URI}" .evergreen/run-tests-serverless.sh
 
+  "stat kmip server":
+    - command: shell.exec
+      params:
+        background: true
+        shell: bash
+        working_dir: "src"
+        script: |
+          source .evergreen/csfle/activate_venv.sh
```

Comment:
`source` is a bashism, the portable equivalent of it is `.` (the dot):

```
. .evergreen/csfle/activate_venv.sh
```

Reply:
Fixed